docs/NSR.md

```diff
@@ -88,7 +88,7 @@ o.	Restrict remote administration or access to an Issuing System, Certificate Ma
  
 Certification Authorities and Delegated Third Parties SHALL:
 
-a.	Implement a Security Support System under the control of CA or Delegated Third Party Trusted Roles that monitors, detects, and reports any security-related configuration change to Certificate Systems;
+a.	Implement a System under the control of CA or Delegated Third Party that continuously monitors, detects, and alerts personnel to any configuration change to Certificate Systems, Issuing Systems, Certificate Management Systems, Security Support Systems, and Front-End / Internal-Support Systems unless the change has been authorized through a change management process.  The CA or Delegated Third Party  shall respond to the alert and initiate a plan of action within at most twenty-four (24) hours.
```

Comment:
```suggestion
a.	Implement a System under the control of CA or Delegated Third Party that continuously monitors, detects, and alerts personnel to any modification to Certificate Systems, Issuing Systems, Certificate Management Systems, Security Support Systems, and Front-End / Internal-Support Systems unless the modification has been authorized through a change management process.  The CA or Delegated Third Party  shall respond to the alert and initiate a plan of action within at most twenty-four (24) hours.
```

This tries to decouple 'change' (as in, the thing tracked in the change management system) from the system state being modified, so that it's clearer any modifications to the system state should have associated changes (proposals, issues, bugs, i.e. "the artifact in a change management system") in the change management system that have followed the change management process.